### PR TITLE
Some quick cleanups for analysis

### PR DESCRIPTION
**Pull request by jeremykubica**
Added some comments to the functions, cleaned up unused imports, and fixed a few warnings (about np.float and np.int being deprecated)

### CODE REVIEW

File: analysis/run_search.py [hunk 1]
```diff
@@ -284,16 +280,19 @@ def run_search(self):
             self.config['im_filepath'], self.config['time_file'],
             self.config['mjd_lims'], self.config['visit_in_filename'],
             self.config['file_format'])
-        # Save values in image_params for later use
+
+        # Apply the mask to the images and set the PDF.
```
Comment (jbkalmbach) [inline analysis/run_search.py:284]: typo, should be PSF

**Review by jbkalmbach — APPROVED** [review]
Thanks for the updates. Just pointed out a couple typos I found.

File: analysis/run_search.py (hunk 1 above)
Comment (jeremykubica) [reply analysis/run_search.py]: Fixed.

**Comment by jeremykubica** [comment]
Fixed both typos. Thanks for catching.

**Pull request merged**